n2y/notion.py

```diff
@@ -366,30 +362,14 @@ def download_file(self, url, page):
         """
         url_path = path.basename(urlparse(url).path)
         _, extension = path.splitext(url_path)
-        with requests.get(url, stream=True) as request_stream:
-            content_iterator = request_stream.iter_content(4096)
-            return self.save_file(content_iterator, page, extension)
+        request_stream = requests.get(url, stream=True)
+        return self.save_file(request_stream.content, page, extension)
 
-    def save_file(self, content_iterator, page, extension):
-        """
-        Save the content in the provided iterator into a file in MEDIA_ROOT. The
-        file name is determined from the page name, file extension, and an md5
-        hash of the content. The md5 hash is calculated as the data is streamed
-        to a temporary file, which is then moved to the final location once the
-        md5 hash can be calculated.
-        """
-        temp_fd, temp_filepath = tempfile.mkstemp()
-        hash_md5 = hashlib.md5()
-        with os.fdopen(temp_fd, 'wb') as temp_file:
-            for chunk in content_iterator:
-                hash_md5.update(chunk)
-                temp_file.write(chunk)
-
-        num_hash_characters = 8  # just long enough to avoid collisions
-        hash = hash_md5.hexdigest()[:num_hash_characters]
-        relative_filepath = "".join([page.filename, "-", hash, extension])
+    def save_file(self, content, page, extension):
+        page_id_chars = page.notion_id.replace('-', '')
```

Comment:
Maybe use https://github.com/innolitics/n2y/blob/main/n2y/utils.py#L126 here? It's just one line, so maybe it's not really necessary to use the function.